blockchain/fullblocksstakeversion_test.go

```diff
@@ -16,7 +16,11 @@ import (
 // enforced properly.
 func TestStakeVersion(t *testing.T) {
 	// Create a test harness initialized with the genesis block as the tip.
-	params := chaincfg.RegNetParams()
+	params := cloneParams(chaincfg.RegNetParams())
+
+	// Simulate an activated DCP0001 by removing its deployment entry.
+	delete(params.Deployments, 5)
```

Comment:
I don't see any issue with this, but it seems unnecessary since the generator doesn't check deployments (it just always uses the new algorithm) and the paths that check `chaincfg.VoteIDSDiffAlgorithm` in the underlying chain are not tested as part of this.

Reply:
Agreed.  Removed.